### PR TITLE
Add returning to update_command

### PR DESCRIPTION
**Pull request by cymed**
This PR adds the "returning" variable available for the INSERT command to UPDATE as well

### CODE REVIEW

File: pirogue/utils.py [hunk 1]
```diff
@@ -440,7 +443,8 @@ def value(col):
 
     return """UPDATE {s}.{t}{a} SET
 {indent}    {cols}
-{indent}  WHERE {where_clause};""".format(
+{indent}  WHERE {where_clause}
+{indent}  ){returning};""".format(
```
Comment (3nids) [inline pirogue/utils.py:447]: there is an extra `)` here or I am missing something?

**Review by 3nids — COMMENTED** [review]
could you add a test for this?

File: pirogue/utils.py (hunk 1 above)
Comment (cymed) [reply pirogue/utils.py:447]: you are correct

**Comment by cymed** [comment]
> could you add a test for this?

So far there are no tests for all the utils, only for the inheritances

**Review by 3nids — APPROVED** [review]

**Pull request merged**